_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
Following #3666, can you rename this file to `otel_logs_source.md`?

Reply:
@dlvenable Sure!

Reply:
@Naarcha-AWS @hdhalter Can/should we do this?

Reply:
@carolxob , Let's not worry about renaming the files if it is a concern. We can address that later.